code/TemplateStudioForWinUICs/Templates/_comp/MT/Project.MenuBar/Views/ShellPage.xaml

```diff
@@ -21,10 +21,14 @@
         </SplitView.Pane>
         <Grid>
             <Grid.RowDefinitions>
-                <RowDefinition Height="Auto" />
+                <RowDefinition Height="32" />
+                <RowDefinition Height="48" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
-            <MenuBar VerticalAlignment="Top">
+            <Grid Grid.Row="0" x:Name="AppTitleBar">
+                <TextBlock x:Name="AppTitleBarText" Text="App Title" VerticalAlignment="Center" TextWrapping="NoWrap" Style="{StaticResource CaptionTextBlockStyle}" Margin="28, 0, 0, 0"/>
```

Comment:
We should add an icon here as shown in https://docs.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui3#title-bar-content-and-drag-regions. There may be one in the repo we can use.

Reply:
We use an Assets folder already when MSIX is enabled. We should put this in the Assets folder rather than Images.